english/find-exit/into-forest/talk-to-cougar/talk-to-cougar.md

```diff
@@ -14,8 +14,6 @@ You ask Sara how she can speak to humans
 
 She says "Be careful asking questions you don't already know the answer to"
 
-You decide you in fact do not want to know the answer to the question,
+[Head back to the forest](../forest.md)
+[Press Sara for an answer](press-for-answer/press-for-answer.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.